uldlib/frontend.py

```diff
@@ -47,17 +47,19 @@ def run(self, parts: List[DownloadPart], stop_event: threading.Event, terminate_
 
 class ConsoleFrontend(Frontend):
     cli_initialized: bool
+    show_parts_progress: bool
 
     last_log: Tuple[str, LogLevel]
 
     last_captcha_log: Tuple[str, LogLevel]
     last_captcha_stats: Dict[str, int]
 
-    def __init__(self):
```

Comment:
Type hint for `args` missing. Also I think it would be better not passing whole `argparse.Namespace` to the Frontend.

Something like this looks better for me:
```python
def __init__(self, show_parts: bool = False):
    ...
    self.show_parts = show_parts
```
```

Reply:
Good catch, thanks I fixed it by this commit
b53ad696fc54c18ec1ee41dfdcd95f1d331419bc